astropy/io/ascii/ui.py

```diff
@@ -175,9 +175,18 @@ def read(table, guess=None, **kwargs):
         List of names to include in fill_values (default= ``None`` selects all names)
     fill_exclude_names : list
         List of names to exclude from fill_values (applied after ``fill_include_names``)
-    fast_reader : bool
+    fast_reader : bool or dict
         Whether to use the C engine, can also be a dict with options which default to ``False``
-        (default= ``True``)
+        (default= ``True``); parameters for options dict:
+
+        use_fast_converter: bool
```

Comment:
This is processed to
<p><strong>fast_reader</strong> : bool or dict</p>
<blockquote>
<div><p>Whether to use the C engine, can also be a dict with options which default to <tt class="docutils literal"><span class="pre">False</span></tt>
(default= <tt class="docutils literal"><span class="pre">True</span></tt>); parameters for options dict:</p>
<dl class="docutils">
<dt>use_fast_converter: bool</dt>
<dd><p class="first last">enable faster but slightly imprecise floating point conversion method</p>
</dd>
<dt>parallel: bool or int</dt>
<dd><p class="first last">multiprocessing conversion using <tt class="docutils literal"><span class="pre">cpu_count()</span></tt> or <tt class="docutils literal"><span class="pre">'number'</span></tt> processes</p>
</dd>
<dt>exponent_style: str</dt>
<dd><p class="first last">One-character string defining the exponent or <tt class="docutils literal"><span class="pre">'Fortran'</span></tt> to auto-detect
Fortran-style scientific notation like <tt class="docutils literal"><span class="pre">'3.14159D+00'</span></tt> (<tt class="docutils literal"><span class="pre">'E'</span></tt>, <tt class="docutils literal"><span class="pre">'D'</span></tt>, <tt class="docutils literal"><span class="pre">'Q'</span></tt>),
all case-insensitive; default <tt class="docutils literal"><span class="pre">'E'</span></tt>, all other imply <tt class="docutils literal"><span class="pre">use_fast_converter</span></tt></p>
</dd>
</dl>
</div></blockquote>

which looks readable in Safari and Firefox (took the extra line at the top to make this work and stop Sphinx from complaining, which is a little bit annoying in `help()`, but I think one could live with it - see also taldcroft's comment above).
Would you suggest all keys in flowing text without line breaks, or did I misunderstand something?